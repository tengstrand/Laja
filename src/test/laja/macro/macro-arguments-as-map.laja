
#macro person(map)
  map.class().name = {map.class().name}
  map.name = {map.name}
  map.age = {map.age}##
#end


#write "target/test-output/macro-arguments-as-map.out"
{person(name: "Homer" age: 45)}
#end
