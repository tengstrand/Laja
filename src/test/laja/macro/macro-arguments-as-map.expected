  map.class().name = java.util.LinkedHashMap
  map.name = Homer
  map.age = 45
